### DISCUSSION TITLE
'pointerdown' fires twice on Apple pencil & S Pen on Galaxy Note

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7720] [Q&A]
https://user-images.githubusercontent.com/87828192/129937922-c543e7f5-7b3d-4a98-bc30-b69f0311b99d.mp4

Please watch this video.
Is it a bug or normal?

I'm using Sprite's on.('pointerdown')

It works well with finger & mouse event,
but when I'm using Apple Pencil and S Pen Galaxy Note, it triggers twice.

**Comment by miltoncandelero** [comment]
Blind shot: is your method called `pointedown`? If a method is called exactly like an event it will be called automagically.

**Reply by humbleCoder1990** [reply]
My code is like this.

```
const sprite = new Sprite(Texture.WHITE)

sprite.interactive = true

let counter = 0

sprite.on('pointerdown', () => {

 counter++
console.log(counter)

})
```

**Comment by humbleCoder1990** [comment]
I found Apple Pencil & S Pen triggers 'mousedown' & 'touchdown' both..
